### PR TITLE
Create database from  a diagram

### PR DESCRIPTION
**Pull request by andym80**
## **Hello, Good Reviewer :wave:** 

We created a database parsed from this **diagram**. 
clinic_diagram
This database has 6 tables: 

-Patients
-Medical_histories
-Treatments
-Invoice_items
-Invoice
- medical_history_has_treatments

Any suggestions are welcome. 

@Aseadd 
@andym80

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE clinic;
+CREATE TABLE patients (id INT PRIMARY KEY, name varchar(50), date_of_bith date);
+CREATE TABLE medical_histories (id INT PRIMARY KEY,admitted_at TIMESTAMP, patient_id INT,status VAR
+CHAR(20));
+CREATE TABLE treatments (id INT PRIMARY KEY,type VARCHAR(25),name VARCHAR(50));
+CREATE TABLE invoice_items (id INT PRIMARY KEY, unit_price DECIMAL,quantity INT,total_price DECIMAL,
+invoice_id INT,treatment_id INT);
+CREATE TABLE invoices (id INT PRIMARY KEY, total_amount DECIMAL, generated_at TIMESTAMP,payed_at TI
+MESTAMP,medical_history_id INT);
+
```
Comment (CollinsTatang) [inline schema_based_on_diagram.sql:10]: Great Job so @andym80 and @Aseadd 

- [ ] Please all "join table" for the many-to-many relationships created, kindly create a table between medical_histories and treatments.
```sql
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ALTER TABLE medical_histories ADD CONSTRAINT pk_patient_id FOREIGN KEY(patient_id) REFERENCES  patients(id);
+ALTER TABLE invoices ADD CONSTRAINT pk_medical_history_id FOREIGN KEY(medical_history_id) REFERENCES  medical_histories(id);
+ALTER TABLE invoice_items ADD CONSTRAINT pk_invoice_id FOREIGN KEY(invoice_id) REFERENCES  invoices(id);
+ALTER TABLE treatments ADD COLUMN medical_histories_id INT;
+ALTER TABLE treatments ADD CONSTRAINT pk_medical_histories_id FOREIGN KEY(medical_histories_id) REFERENCES medical_histories(id);
+ALTER TABLE treatments ADD COLUMN invoice_items_id INT;
+ALTER TABLE treatments ADD CONSTRAINT pk_invoice_items_id FOREIGN KEY (invoice_items_id) REFERENCES invoice_items(id);
+
+
+
```
Comment (CollinsTatang) [inline schema_based_on_diagram.sql:34]: - [ ] Please all FOREIGN KEYS have indexes created
* `CREATE INDEX ON medical_histories (patient_id);`
* `CREATE INDEX ON invoices (medical_history_id);`
* `CREATE INDEX ON invoice_items (invoice_id);`
* `CREATE INDEX ON invoice_items (treatment_id);`
* `CREATE INDEX ON medical_histories_has_treatments (medical_history_id);`
* `CREATE INDEX ON medical_histories_has_treatments (treatment_id);`

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @andym80 and @Aseadd , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @andym80, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**